StripeUICore/StripeUICore/Source/Elements/PhoneNumber/PhoneNumberElement.swift

```diff
@@ -94,6 +94,13 @@ import UIKit
         self.textFieldElement.delegate = self
     }
 
+    public func setSelectedCountryCode(_ countryCode: String, shouldUpdateDefaultNumber: Bool = false) {
```

Comment:
Why is the default value for `shouldUpdateDefaultNumber` both here and in `selectCountry(index:shouldUpdateDefaultNumber:)` false when it's only ever called with true except for in one test?

Reply:
I don't know why the original was done that way, but IMO it makes semantical sense for the default to be false, if you call `setSelectedCountryCode("US")`, I wouldn't expect it to have additional side effects, changing the default number in this case, so it makes sense that you have to explicitly signal that you want the side effect.

Reply:
I guess my question is more: Why have a default value at all if we're not actually using it?

Reply:
I don't think there is a compelling reason to have it or not have it, but in general I prefer to add defaults when there is a sensible one, even if is not used at the moment it could be useful later, and it signals what's the "default" way to use an API.

Reply:
If we can think of a scenario where we'd want to update the country code without changing the default number in the future, and that that would be the common/default way to call the API, I'm fine keeping it. It just doesn't seem like we have a use case for that, which implies just making it a regular parameter without a default